player/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.conf import settings
+from meet_balls_app.enums import BallGame
+
+
+class Player(models.Model):
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        primary_key=True
+        )
```

Comment:
We're going to need unit tests to have this merged, as it includes integration with the built in user model and we need to see that the built in user is created/deleted/updated as expected.